lib/containers/utils.pm

```diff
@@ -108,96 +108,96 @@ sub basic_container_tests {
     my $tumbleweed           = "registry.opensuse.org/opensuse/tumbleweed";
 
     # Test search feature
-    validate_script_output("$runtime search --no-trunc tumbleweed", sub { m/Official openSUSE Tumbleweed images/ });
+    validate_script_output("$runtime->{runtime} search --no-trunc tumbleweed", sub { m/Official openSUSE Tumbleweed images/ });
 
     #   - pull minimalistic alpine image of declared version using tag
     #   - https://store.docker.com/images/alpine
-    assert_script_run("$runtime image pull $alpine", timeout => 300);
+    assert_script_run("$runtime->{runtime} image pull $alpine", timeout => 300);
```

Comment:
Don't we have `$runtime->pull()`?

Reply:
as i said i havent touch some places intentional. tbd

Reply:
OK. I see. This won't break the tests and it can be easily done in follow up commit.